Library/Formula/duplicity.rb

```diff
@@ -319,6 +319,8 @@ def install
       r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
     end
 
+    chmod 0644, Dir[vendor_site_packages/"prettytable-*-py*.egg-info/*"]
```

Comment:
The pip does not really installs the same files:

PIP:
````
-rw-r--r--  1 _brew  _brew     10 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/DESCRIPTION.rst
-rw-r--r--  1 _brew  _brew    700 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/METADATA
-rw-r--r--  1 _brew  _brew    606 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/RECORD
-rw-r--r--  1 _brew  _brew     92 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/WHEEL
-rw-r--r--  1 _brew  _brew    821 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/metadata.json
-rw-r--r--  1 _brew  _brew     12 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/top_level.txt
-rw-r--r--  1 _brew  _brew  54204 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable.py
-rw-r--r--  1 _brew  _brew  52667 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable.pyc
````

The duplicity install:

````
-rw-------  1 _brew  _brew    710 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/PKG-INFO
-rw-------  1 _brew  _brew    230 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/SOURCES.txt
-rw-------  1 _brew  _brew      1 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/dependency_links.txt
-rw-------  1 _brew  _brew     12 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/top_level.txt
-rw-r--r--  1 _brew  _brew  54204 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable.py
````

As you can see the install is not quite the same. In the duplicity install at leat the PKG-INFO shoulf be readable.